src/rust/Cargo.toml

```diff
@@ -14,5 +14,8 @@ cxx = "1.0"
 im-rc = "15.0.0"
 base64 = "0.13.0"
 rustc-simple-version = "0.1.0"
-soroban-env-host = { version = "0.0.10", features = [ "vm" ] }
-soroban-test-wasms = { git = "https://github.com/stellar/rs-soroban-env", rev = "c1480516" }
+# soroban-env-host = { version = "0.0.10", features = ["vm"] }
```

Comment:
Is this right? I thought we could specify the version and the update git ref like we do [here](https://github.com/stellar/rs-soroban-env/blob/249f6597eb125255133af9ce9a1f5daa6388d254/Cargo.toml#L27). cc @leighmcculloch

Reply:
Yeah there's no need to remove the version. Leaving the version in also makes sure you're using a commit that has that version in its .toml still too.

Reply:
When I just update the version number `soroban-env-host = { version = "0.0.11", features = ["vm"] }`
it still picks up the old code, and it would not let me specify both `version` and `rev` together.

Reply:
This appears to work 
`soroban-env-host = { git = "https://github.com/stellar/rs-soroban-env", rev = "85cabe5", version = "0.0.11", features = [
    "vm",
] }`
@leighmcculloch Does this look right to you?

Reply:
👍🏻 

For the sake of making it easy to comment out lines as you need them, you may want to use the toml style that we use in the env:

```toml
[workspace.dependencies.stellar-xdr]
version = "0.0.11"
git = "https://github.com/stellar/rs-stellar-xdr"
rev = "dbf2aba"
default-features = false
```

Reply:
Thanks, fixed it!

Reply:
I noticed you added a version to `soroban-test-wasms` as well, which it didn't have before. I think the version wasn't specified initially because `soroban-test-wasms` isn't published, so I think it should be removed.

Reply:
Got it, removed